lib/network/modules/Canvas.js

```diff
@@ -374,7 +372,7 @@ class Canvas {
   _determinePixelRatio() {
     let ctx = this.getContext();
     if (ctx === undefined) {
-     throw "Could not get canvax context";
+     throw Error("Could not get canvax context");
```

Comment:
**Finally** something I can regard as an honest review statement.
Missing `new`:
```js
  throw new Error("disaster!");
```

Goes for all other occurrences as well.

Reply:
Please change this and similar; this is code and definitely wrong.